hudi-client/hudi-client-common/src/test/java/org/apache/hudi/utils/HoodieWriterClientTestHarness.java

```diff
@@ -1171,19 +1180,24 @@ protected void testHoodieConcatHandleOnDupInserts(boolean isPrepped, InstantGene
    */
   protected void testUpsertsInternal(Function3<Object, BaseHoodieWriteClient, Object, String> writeFn, boolean populateMetaFields, boolean isPrepped,
                                      SupportsUpgradeDowngrade upgradeDowngrade) throws Exception {
-    metaClient = createMetaClient();
-    HoodieWriteConfig.Builder cfgBuilder = getConfigBuilder(HoodieFailedWritesCleaningPolicy.LAZY).withRollbackUsingMarkers(true)
-        .withMetadataConfig(HoodieMetadataConfig.newBuilder().enable(false).build());
-    addConfigsForPopulateMetaFields(cfgBuilder, populateMetaFields);
-    // Force using older timeline layout
-    HoodieTableMetaClient.newTableBuilder()
+
+    metaClient.getStorage().deleteDirectory(new StoragePath(basePath));
```

Comment:
is the path shared among multiple test cases?

Reply:
nope. in general, we have a set up method before each method which will set up the random base path and set up the meta client too. in this test, looks like we are starting w/ version 6 (L1186 to 1190). and hence to be clean, I am deleting everything under base path and starting the test. 

this is not required for other tests which might just work w/ table version 8.